docs/docs/api-changes.md

```diff
@@ -43,7 +43,7 @@ they are _not_ thread-safe.
 
 Conclave Init now requires Java 17 to run, and generated projects will target Java 17 by default.
 
-Conclave SDK version is now set at the individual project level, rather at the OS user
+The SDK version is now set at the individual project level, rather at the OS user
```

Comment:
The SDK version is now set at the individual project level rather than at the OS user level.